### PR TITLE
Adding SignBERT+

### PR DESCRIPTION
**Pull request by cleong110**
Adding SignBERT+ summary

Paper: https://ieeexplore.ieee.org/document/10109128

Detailed description of summary process: https://github.com/cleong110/sign-language-processing.github.io/issues/14

Summary of PR: 

* Adding the paper and summary
* Adding some key references

**Questions:**
* should this be in the pose-to-text section? That is one of the tasks they evaluate on but it's a pretraining framework for general SLU, and they just validate on it. 
* Would it be good to have a general "pretraining" section instead?
* Added a citation for RWTH-PHOENIX-WEATHER (the 2015 continuous SLR set, koller2015ContinuousSLR, not the translation set!), because it seemed we didn't have one. Do we need an explanation somewhere of the distinction between all the different RWTH sets?
* (lower priority) What do we do about the [HANDS17 dataset](https://github.com/cleong110/sign-language-processing.github.io/issues/14#issuecomment-2134004288) or other similar datasets that aren't specifically for SLU but are useful?  
* What do we do about SignBERT (https://openaccess.thecvf.com/content/ICCV2021/html/Hu_SignBERT_Pre-Training_of_Hand-Model-Aware_Representation_for_Sign_Language_Recognition_ICCV_2021_paper.html)? They describe SignBERT+ as an extension of SignBERT, should we mention it?

### CODE REVIEW

File: src/references.bib [hunk 1]
```diff
@@ -3100,3 +3100,102 @@ @inproceedings{post-2018-call-sacrebleu
     doi = "10.18653/v1/W18-6319",
     pages = "186--191"
 }
+
+@ARTICLE{hu2023SignBertPlus,
+  author={Hu, Hezhen and Zhao, Weichao and Zhou, Wengang and Li, Houqiang},
+  journal={IEEE Transactions on Pattern Analysis and Machine Intelligence}, 
+  title={SignBERT+: Hand-Model-Aware Self-Supervised Pre-Training for Sign Language Understanding},
```
Comment (AmitMY) [inline src/references.bib:3107]: `{SignBERT}`

File: src/references.bib [hunk 2]
```diff
@@ -3100,3 +3100,102 @@ @inproceedings{post-2018-call-sacrebleu
...
+  year={2023},
+  volume={45},
+  number={9},
+  pages={11221-11239},
+  doi={10.1109/TPAMI.2023.3269220}
+}
+
+@InProceedings{hu2021SignBert,
+    author    = {Hu, Hezhen and Zhao, Weichao and Zhou, Wengang and Wang, Yuechen and Li, Houqiang},
+    title     = {SignBERT: Pre-Training of Hand-Model-Aware Representation for Sign Language Recognition},
```
Comment (AmitMY) [inline src/references.bib:3117]: `{SignBERT}`

File: src/references.bib [hunk 3]
```diff
@@ -3100,3 +3100,102 @@ @inproceedings{post-2018-call-sacrebleu
...
+    booktitle = {Proceedings of the IEEE/CVF International Conference on Computer Vision (ICCV)},
+    month     = {October},
+    year      = {2021},
+    pages     = {11087-11096}
+}
+
+
+@article{hu2021NMFAwareSLR,
+	author = {Hu, Hezhen and Zhou, Wengang and Pu, Junfu and Li, Houqiang},
+	title = {Global-Local Enhancement Network for NMF-Aware Sign Language Recognition},
```
Comment (AmitMY) [inline src/references.bib:3127]: `{NMF}`

File: src/references.bib [hunk 4]
```diff
@@ -3100,3 +3100,102 @@ @inproceedings{post-2018-call-sacrebleu
...
+	url = {https://doi.org/10.1145/3436754},
+	doi = {10.1145/3436754},
+	journal = {ACM Trans. Multimedia Comput. Commun. Appl.},
+	month = {jul},
+	articleno = {80},
+	numpages = {19}
+}
+
+@misc{yuan20172017,
+      title={The 2017 Hands in the Million Challenge on 3D Hand Pose Estimation},
```
Comment (AmitMY) [inline src/references.bib:3144]: `{3D}`

File: src/references.bib [hunk 5]
```diff
@@ -3100,3 +3100,102 @@ @inproceedings{post-2018-call-sacrebleu
...
+url = {https://doi.org/10.1145/3130800.3130883},
+doi = {10.1145/3130800.3130883},
+journal = {ACM Trans. Graph.},
+month = {nov},
+articleno = {245},
+numpages = {17}
+}
+
+@misc{mmpose2020,
+    title={OpenMMLab Pose Estimation Toolbox and Benchmark},
```
Comment (AmitMY) [inline src/references.bib:3172]: {OpenMMLab}

File: src/references.bib [hunk 6]
```diff
@@ -3100,3 +3100,102 @@ @inproceedings{post-2018-call-sacrebleu
...
+    author={MMPose Contributors},
+    howpublished = {\url{https://github.com/open-mmlab/mmpose}},
+    year={2020}
+}
+
+@ARTICLE{huang2019attention3DCNNsSLR,
+  author={Huang, Jie and Zhou, Wengang and Li, Houqiang and Li, Weiping},
+  journal={IEEE Transactions on Circuits and Systems for Video Technology}, 
+  title={Attention-Based 3D-CNNs for Large-Vocabulary Sign Language Recognition},
```
Comment (AmitMY) [inline src/references.bib:3181]: {3D-CNNs}

File: src/datasets/NMFs_CSL.json [hunk 7]
```diff
@@ -0,0 +1,16 @@
+{
+    "pub": {
+      "name": "NMFs-CSL",
+      "year": 2021,
+      "publication":"hu2021NMFAwareSLR",
+      "url": "https://ustc-slr.github.io/datasets/2020_nmfs_csl/"
+    },
+    "#loader": null,
+    "#items": 1067,
+    "#samples": "32010 videos",
```
Comment (AmitMY) [inline src/datasets/NMFs_CSL.json:10]: 32,010

File: src/index.md [hunk 8]
```diff
@@ -788,6 +788,14 @@ and showed similar performance, with the transformer underperforming on the vali
 They experimented with various normalization schemes, mainly subtracting the mean and dividing by the standard deviation of every individual keypoint
 either concerning the entire frame or the relevant "object" (Body, Face, and Hand).
 
+@hu2023SignBertPlus introduce SignBERT+, a self-supervised pretraining method for sign language understanding (SLU) incorporating a hand-model-aware approach.
```
Comment (AmitMY) [inline src/index.md:791]: `hand-model-aware` is not a clear compound when reading

File: src/index.md [hunk 9]
```diff
@@ -788,6 +788,14 @@ and showed similar performance, with the transformer underperforming on the vali
...
+They extract pose sequences from over 230k videos using MMPose [@mmpose2020], treating these as visual tokens embedded with temporal positional encodings.
+They pretrain using multi-level masked modeling (joints, frames, clips) and integrate a statistical hand model [@romero2017MANOHandModel] to constrain the decoder's predictions for anatomical realism and enhance its accuracy.
+Validation on isolated SLR (MS-ASL [@dataset:joze2018ms], WLASL [@dataset:li2020word], SLR500 [@huang2019attention3DCNNsSLR].), continuous SLR (RWTH-PHOENIX-Weather [@koller2015ContinuousSLR]), and SLT (RWTH-PHOENIX-Weather 2014T [@dataset:forster2014extensions;@cihan2018neural]) demonstrates state-of-the-art performance.
```
Comment (AmitMY) [inline src/index.md:794]: dot after SLR500 can be removed

File: src/index.md (hunk 8 above)
Comment (AmitMY) [inline src/index.md:791]: Unclear to me how this relates to SignBERT.

Usually it would be:
> .... introduce SignBERT, .....
They then introduce SignBERT+ with better ____ modeling.

File: src/index.md (hunk 9 above)
Comment (AmitMY) [inline src/index.md:794]: Writing is generally good!

**Review by AmitMY — CHANGES_REQUESTED** [review]

File: src/index.md (hunk 9 above)
Comment (cleong110) [reply src/index.md:794]: Nice catch!

File: src/index.md (hunk 8 above)
Comment (cleong110) [reply src/index.md:791]: Yes, I was wondering about this https://github.com/sign-language-processing/sign-language-processing.github.io/pull/54#issue-2321200142. 

tl;dr it's an... "extension of the conference paper" but they added a few things.

image
So I was wondering whether we should try and describe both, or just SignBERT+.

File: src/index.md (hunk 9 above)
Comment (cleong110) [reply src/index.md:794]: Thanks!! :D

File: src/index.md (hunk 8 above)
Comment (cleong110) [reply src/index.md]: Took a crack at weaving it in

Comment (cleong110) [reply src/index.md]: Rewrote to try and make that a bit smoother.

File: src/index.md [hunk 10]
```diff
@@ -803,6 +803,16 @@ and showed similar performance, with the transformer underperforming on the vali
 They experimented with various normalization schemes, mainly subtracting the mean and dividing by the standard deviation of every individual keypoint
 either concerning the entire frame or the relevant "object" (Body, Face, and Hand).
 
+@hu2023SignBertPlus introduce SignBERT+, a self-supervised pretraining method for sign language understanding (SLU) that is aware of and informed by a hand model in its predictions.
```
Comment (AmitMY) [inline src/index.md:806]: convoluted phrasing:
>  that is aware of and informed by

File: src/index.md [hunk 11]
```diff
@@ -803,6 +803,16 @@ and showed similar performance, with the transformer underperforming on the vali
...
+They extract pose sequences from over 230k videos using MMPose [@mmpose2020], treating these as visual tokens.
+This is an extension of their earlier SignBERT [@hu2021SignBert], with several improvements.
+Notably, on SignBERT+, they embed these tokens with temporal positional encodings.
+They then pretrain using multi-level masked modeling (joints, frames, clips) and integrate a statistical hand model [@romero2017MANOHandModel] to constrain the decoder's predictions for anatomical realism and enhance its accuracy.
```
Comment (AmitMY) [inline src/index.md:810]: I feel like these sentences are just fragments, and don't read well as a paragraph.
My quick draft rewrite would be:

```md
@hu2023SignBertPlus introduce SignBERT+, a self-supervised pretraining method for sign language understanding (SLU).
This is an extension of their earlier SignBERT [@hu2021SignBert], with several improvements.
For pertaining, they extract pose sequences from over 230k videos using MMPose [@mmpose2020].
They then perform multi-level masked modeling (joints, frames, clips) and integrate a statistical hand model [@romero2017MANOHandModel] to constrain the decoder's predictions for anatomical realism and enhance its accuracy.
```

**Review by AmitMY — CHANGES_REQUESTED** [review]

File: src/index.md (hunk 10 above)
Comment (cleong110) [reply src/index.md:806]: Agreed, tbh

File: src/index.md (hunk 11 above)
Comment (cleong110) [reply src/index.md]: Took another crack at i.

**Review by AmitMY — APPROVED** [review]
